eng/Versions.props

```diff
@@ -1,32 +1,61 @@
 <Project>
-  <!--Package versions-->
   <PropertyGroup>
+    <!-- dotnet/installer -->
     <MicrosoftDotnetSdkInternalPackageVersion>6.0.100-rc.2.21423.2</MicrosoftDotnetSdkInternalPackageVersion>
+    <!-- dotnet/runtime -->
     <MicrosoftNETCoreAppRefPackageVersion>6.0.0-rc.2.21420.29</MicrosoftNETCoreAppRefPackageVersion>
+    <!-- xamarin/xamarin-android -->
     <MicrosoftAndroidSdkWindowsPackageVersion>31.0.100-rc.2.12</MicrosoftAndroidSdkWindowsPackageVersion>
-    <MicrosoftMacCatalystSdkPackageVersion>15.0.100-rc.1.521</MicrosoftMacCatalystSdkPackageVersion>
+    <!-- xamarin/xamarin-macios -->
```

Comment:
I tried to put important values at the top here with comments, and alphabetically sorted the rest.